api/src/services/cards.service.ts

```diff
@@ -69,5 +102,14 @@ export class CardService {
 
     return createdPriceHistory;
   }
+
+  public async createCardRating(cardId : ObjectId | string , cardRatingData: CardRating): Promise<CardRating> {
```

Comment:
cardId is declared but never used, not sure if you need a cardId. I'm guessing the 'cardRatingData' contains the cardId. Also I noticed the same thing with the above function.

Reply:
Oh yes, you are right. The `cardId` had already been injected into the `cardRatingData` in the controller. Nice catch